### PR TITLE
(DOCSP-20975): [Admin API] HTTPS Endpoints

### PR DESCRIPTION
**Pull request by nlarew**
### Jira

- (DOCSP-20975): [Admin API] HTTPS Endpoints

### Staged Changes (Requires MongoDB Corp SSO)

- [Custom HTTPS Endpoint APIs](https://docs-mongodbcom-staging.corp.mongodb.com/realm/docsworker-xlarge/endpoints-admin/admin/api/v3/#tag/endpoints)

### Review Guidelines

[REVIEWING.md](https://github.com/mongodb/docs-realm/blob/master/REVIEWING.md)

### CODE REVIEW

File: source/openapi-admin-v3.yaml [hunk 1]
```diff
@@ -2876,6 +2876,105 @@ paths:
       responses:
         "204":
           description: No Content
+
+
+  "/groups/{groupId}/apps/{appId}/endpoints":
```
Comment (mongodben) [inline source/openapi-admin-v3.yaml:2881]: how meta to have an api endpoint abt creating api endpoints 😂

File: source/openapi-admin-v3.yaml [hunk 2]
```diff
@@ -2876,6 +2876,105 @@ paths:
...
+      description: Get a specific [endpoint](https://docs.mongodb.com/realm/endpoints/) configuration.
+      responses:
+        "200":
+          description: Ok
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  "$ref": "#/components/schemas/Endpoint"
```
Comment (mongodben) [inline source/openapi-admin-v3.yaml:2900]: i think this is the same content as the stuff on lines 2923 - 2941. i believe it should be removed, unless this endpoint has a different use, like get **ALL** endpoints

Comment (mongodben) [reply source/openapi-admin-v3.yaml:2900]: also the operationId is the same, `adminGetEndpoint`

File: source/openapi-admin-v3.yaml [hunk 3]
```diff
@@ -2876,6 +2876,105 @@ paths:
...
+          content:
+            application/json:
+              schema:
+                "$ref": "#/components/schemas/Endpoint"
+    
+    put:
+      tags:
+        - endpoints
+      operationId: adminModifyEndpoint
+      summary: Modify an endpoint.
```
Comment (mongodben) [inline source/openapi-admin-v3.yaml:2947]:
```suggestion
      summary: Modify an endpoint
```

File: source/openapi-admin-v3.yaml (hunk 2 above)
Comment (nlarew) [reply source/openapi-admin-v3.yaml:2900]: Oops good catch, it should be for all endpoints. Updated!

**Review by mongodben — APPROVED** [review]
lgtm

**Pull request merged**